### PR TITLE
Advice

### PR DESCRIPTION
**Pull request by kassiebartow**

### CODE REVIEW

**Comment by twosavoie** [comment]
Hi @kassiebartow,

This is great! Well done!

Only one small change to make before we can merge your code into the Skillcrush repo. Can you remove the conflict markers in the index.html file around the link to your advice file? 

Once you have, please add the change with `git add -A`, commit as per usual and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Best,
Lisa

File: index.html [hunk 1]
```diff
@@ -1318,7 +1318,11 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/shanna-walsh-advice.html" target="_blank">Shanna's Advice</a><br>
       <a href="advice/chris-sinclair-advice.html" target="_blank">Chris Sinclair's Advice</a><br>
       <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>
+<<<<<<< HEAD
+      <a href="advice/kassie-bartow-advice.html" target="_blank">Kassie's Advice 🦋</a><br>
+=======
       <a href="advice/april-jordan-advice.html" target="_blank">April's Advice</a><br>
+>>>>>>> main
```
Comment (twosavoie) [inline index.html:1325]: Hi @kassiebartow,

It looks like the conflict markers haven't been removed yet. If you've made this change already, check that your code has been saved and then add and commit the change and push your advice branch to GitHub. 

Let us know if you have any questions! 

Best,
Lisa

**Comment by kassiebartow** [comment]
I'm not seeing conflict markers anymore on my end. I went ahead and
resaved, added, committed, and pushed again anyway. I'm including
screenshots to see where I might be lost. Thanks for your help! [image:
Screen Shot 2023-11-01 at 7.14.31 AM.png][image: Screen Shot 2023-11-01 at
7.18.30 AM.png]

**Comment by redrambles** [comment]
Hey @kassiebartow !

The screenshots aren't coming through here - can you please send them on over to questions@skillcrush.com? We'll go from there!

Cheers,
Ann :)

**Comment by redrambles** [comment]
Hey @kassiebartow !

Well done - your code is merged in! 🎉

Cheers,
Ann :)

**Pull request merged**